### PR TITLE
java / py sols for cf-1879D

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

File: solutions/silver/cf-1879D.mdx [hunk 1]
```diff
@@ -87,6 +87,90 @@ int main() {
...
 
 </CPPSection>
+
+<JavaSection>
+
+```java
+import java.util.*;
+
+public class SumofXORFunctions {
+	static int MOD = 998244353;
```
Comment (SansPapyrus683) [inline solutions/silver/cf-1879D.mdx:97]: 1. rename class to `SumOfXOR`
2. make the mod final

File: solutions/silver/cf-1879D.mdx [hunk 2]
```diff
@@ -87,6 +87,90 @@ int main() {
...
+```
+
+</JavaSection>
+
+<Pysection>
+
+```py
+MOD = 998244353
+n = int(input())
+a = [*map(int, input().split())]
```
Comment (SansPapyrus683) [inline solutions/silver/cf-1879D.mdx:143]:
```suggestion
a = [int(i) for i in input().split()]
```

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: solutions/silver/cf-1879D.mdx (hunk 1 above)
Comment (freakin23) [reply solutions/silver/cf-1879D.mdx]: done.

**Review by SansPapyrus683 — APPROVED** [review]
show proof of ac for each again, since some changes were made

**Comment by freakin23** [comment]
> show proof of ac for each again, since some changes were made

AC (JAVA): https://codeforces.com/contest/1879/submission/278420088
AC (Python): https://codeforces.com/contest/1879/submission/278420029

**Pull request merged**